ydb/core/tx/datashard/prefix_kmeans.cpp

```diff
@@ -102,15 +102,19 @@ class TPrefixKMeansScanBase: public TActor<TPrefixKMeansScanBase>, public NTable
 
     ui32 PrefixColumns;
     TSerializedCellVec Prefix;
-    bool HasNextPrefix = false;
+    TBufferData PrefixRows;
+    bool IsFirstPrefixFeed = true;
+    bool IsPrefixRowsValid = true;
```

Comment:
I think you can easily extend current tests to test both behavior (now only small prefixes tested):
1. Lets add max rows count in prefix kmeans proto message
2. Run prefix_kmeans tests with 0 and 1000 max rows count

Reply:
Ok, will do separately https://github.com/orgs/ydb-platform/projects/32/views/7?pane=issue&itemId=104049167

And better add limits